src/WebformCivicrmPostProcess.php

```diff
@@ -528,14 +528,14 @@ private function validateParticipants() {
         'return' => ["id"],
         'name' => "Event Fee",
         'is_active' => 1,
-      ])[0];
+      ])[0]['id'];
```

Comment:
@briennekordis Can we update this line to `], 'id')[0] ?? NULL;` to safeguard it from e-notices? With the above code, I get the following if there is no `Event Fee` financial type in civicrm.

>Notice: Undefined offset: 0 in Drupal\webform_civicrm\WebformCivicrmPostProcess->validateParticipants() (line 531 of modules/contrib/webform_civicrm/src/WebformCivicrmPostProcess.php)

Reply:
@jitendrapurohit Sure, I just pushed a commit adding the null coalescing operator.